README.md

```diff
@@ -22,14 +22,18 @@ To learn more, see the [Cluster API KEP][cluster-api-kep].
 
 * Chat with us on [Slack](http://slack.k8s.io/): #cluster-api
 
-* Pointers to repositories and PRs where some Cluster API provisioners are being
-  developed.
+## Provider Implementations
+
+The code in this repository is independent of any specific deployment environment.
+Provider specific code is being developed in separate repositories, some of which
+are also sponsored by SIG-cluster-lifecycle:
+
   * AWS, https://github.com/kubernetes-sigs/cluster-api-provider-aws
   * AWS/Openshift, https://github.com/openshift/cluster-operator
   * Azure, https://github.com/platform9/azure-provider
   * GCE, https://github.com/kubernetes-sigs/cluster-api-provider-gcp
   * OpenStack, https://github.com/kubernetes-sigs/cluster-api-provider-openstack
-  * vSphere, https://github.com/kubernetes-sigs/cluster-api/tree/master/cloud/vsphere
+  * vSphere, https://github.com/roberthbailey/cluster-api-provider-vsphere
```

Comment:
we fix this later right when the application for vsphere repo goes through... which is fine

Reply:
yes. Since the code is no longer in this repo I wanted to point somewhere where it actually exists today. I think our option are to either find it a permanent home or stop linking to it.